doc/classes/DirAccess.xml

```diff
@@ -104,6 +104,17 @@
 				[b]Note:[/b] This method is implemented on macOS, Linux, and Windows.
 			</description>
 		</method>
+		<method name="create_temp" qualifiers="static">
+			<return type="DirAccess" />
+			<param index="0" name="prefix" type="String" default="&quot;&quot;" />
+			<param index="1" name="keep" type="bool" default="false" />
+			<description>
+				Creates a temporary directory.
+				If [param prefix] is not empty, the directory name will be preceded by the [String] you passed, separated by a [code]-[/code].
+				If [param keep] is [code]true[/code], the directory will outlive the [DirAccess] instance. When [code]false[/code], the directory will be deleted, including all its files.
```

Comment:
Perhaps...?
```suggestion
				If [param keep] is [code]true[/code], the directory is not deleted when the returned [DirAccess] is freed. Otherwise, as by default, the directory will be deleted, including all its files.
```
Or, if the suggestion on the first line is accepted, there would be no need to explain this
```suggestion
				If [param keep] is [code]true[/code], the directory is not deleted when the returned [DirAccess] is freed.
```
Or similar.

Reply:
I chose the second suggestion here.